package/tests/test_PartSeg/conftest.py

```
@@ -90,7 +90,7 @@ def start(self, time=None):
             else:
                 old_start(self)
 
-    def not_start(self):
+    def not_start(self, time=None):
```

Ruff (0.14.14)

[warning] Unused function argument: `self`

(ARG001)